cli/route.go

```diff
@@ -38,15 +45,23 @@ Options:
 	--no-drain-backends        don't wait for in-flight requests to complete before stopping backends
 	--disable-keep-alives      disable keep-alives between the router and backends for the given route
 	--enable-keep-alives       enable keep-alives between the router and backends for the given route (default for new routes)
+	-f, --file=<file>          name of file to write generated config to (defaults to stdout)
+	--force                    forcibly apply route changes without user confirmation
 
 Commands:
 	With no arguments, shows a list of routes.
 
+	config  generates or applies route config
 	add     adds a route to an app
+	update  updates a route
 	remove  removes a route
 
 Examples:
 
+        $ flynn route config generate -f routes.cfg app1 app2 app3
+
+        $ flynn route config apply routes.cfg
```

Comment:
The indentation appears to be a tiny but off for these compared to the other examples below

Reply:
Good spot 👀 Fixed in 8b4c8a59.